src/map.cpp

```diff
@@ -57,6 +64,22 @@ int map_set(Map *map, int x, int y, int z, int w) {
         index = (index + 1) & map->mask;
         entry = map->data + index;
     }
+    if(w == 16) //if Cloud
+    {
+        // std::vector<cloudPosition> allClouds;
+        printf("There are Cloud Blocks!?!?!\n");
```

Comment:
Just wanted to point out that there's a lot of commented out code here. Also, this print statement is kind of uninformative.

We need to figure out how we want to do testing. I'm okay with print statement testing, but we'll probably have to come up with a format.

Reply:
This is just an artifact of me trying to begin implementing the cloud functions. I decided that it would be best to comment it out for now - since the code was stable. I'll be working on the implementation in this same spot in the code in the future so I don't think there is a problem with leaving it here for now.

Reply:
I am currently building tests into the cloud.h/cloud.cpp files to start off with. But we will need to settle on a testing method - this seems like a topic for a meeting though.